src/views/List.jsx

```diff
@@ -1,23 +1,41 @@
+import { useState } from 'react';
 import { ListItem } from '../components';
 
 export function List({ data }) {
+	const [searchTerm, setSearchTerm] = useState('');
+
+	/* TO DO: Make separate resuable input component with a filter feature*/
+
+	const filterList = data.filter((item) => {
+		if (searchTerm === '') {
+			return item;
+		} else if (item.name.toLowerCase().includes(searchTerm.toLowerCase())) {
+			return item;
+		}
+	});
+
+	const renderList = filterList.map((item) => (
+		<ListItem name={item.name} key={item.id} />
+	));
+
+	const handleSubmit = (e) => {
```

Comment:
Nit: are we actually submitting anything here? Perhaps we could rename this function to something like `clearSearchField`. Totally up to you though!